### PR TITLE
[WIP] Adapt EMCAL trigger table to STU format in run3

### PR DESCRIPTION
**Pull request by mfasDa**
- Only 2 columns (FastOR ID and ADC)
- All FastOR ADCs only for fraction of events (settable)
- FastOR ADCs from the 3 leading patches for the other events

### CODE REVIEW

**Comment by mfasDa** [comment]
Work in progress - don't merge yet!

**Comment by mfasDa** [comment]
Test could be launched.

**Review by jgrosseo — CHANGES_REQUESTED** [review]
(copied from mail exchange) Your PR makes a new table for the emcal trigger, but we agreed to have both in the same table, didn't we? Wouldn't it be better you adjust the calotrigger table as suggested by Dimitri and keep the filling there?

**Comment by jgrosseo** [comment]
Needs merging with phos changes first

**Comment by mfasDa** [comment]
There was no reason to close this pull request!

Indeed as I mentioned in the private discussion the developments from PHOS and EMCAL are parallel and therefore conflicting, however I would have rebased the branch and update the PR once the PHOS changes are merged. Also therefore the PR was marked as [WIP].

Please for the future keep the pull requests open!

**Pull request closed without merging**
